.github/workflows/changelog.yml

```diff
@@ -0,0 +1,37 @@
+name: "Generate Changelog and create PR"
+
+on:
+    workflow_dispatch:
+        inputs:
+            version:
+                description: 'The version that is going to be released. E.g. 1.22.0'
+                required: true
+            release_date:
+                description: 'The date of the release (YYYY-MM-DD). E.g. 2021-06-14'
+                required: true
+            amount_of_versions:
+                description: 'THe amount of versions to generate changelog for. E.g. 12'
```

Comment:
What does it mean amount of versions? If I want to release 1.23.0 and now the version is 1.22.8 then should I put 8 here?